cabal.project

```diff
@@ -168,78 +168,78 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger-specs
-  tag: 6611345670dad971fbd03594a00b5ef8afa4cae6
-  --sha256: 0rhdlf7kz62231127ywbakhsgsxhhkba5gz24dfy7l9r45a1ijkk
+  tag: 47e2075f751d8370222a2aad32f37699a341ec40
```

Comment:
I was planning to update the dependency on cardano-ledger-specs today, now that https://github.com/input-output-hk/cardano-ledger-specs/pull/1664 is merged.

Is that alright or does that perhaps breaking changes that are not wanted?

Reply:
@nc6 is working on this already.

Reply:
More than one update is fine. Yes we'll need to do it again to get the latest ledger changes.